src/routes/community-hub/+page.svelte

```diff
@@ -456,6 +456,17 @@
 
     transition: background-color 0.15s;
 
+    &::after {
```

Comment:
I guess this is how the caret down is put there now, but even after understanding, that it's inserted like this I sincerly don't understand how that is controlled or specified further. So I'm not sure if this is good for making it easy to maintain.

How does that work and is there a good reason for that to be done this way?

Reply:
It uses borders to draw a triangle. I copied it from the other dropdowns such as the ones in the navbar and the search dropdown. I initially used this because the FA icon would be way too big when first loading the page, before resizing a little later.

![chrome_l3HgXskgxv](https://github.com/beastsaber/bsaber/assets/73407947/e26fe76c-cd6c-4311-be99-33cb31ff7342)

Reply:
I see, it's that trick. It flew right past me. That makes sense.